exercises/ex3.rb

```diff
@@ -0,0 +1,48 @@
+class Geometry
+  attr_accessor :x, :y
+
+  def initialize(x, y)
+    @x = x
+    @y = y
+  end
+
+  def distance(point)
+    Math.sqrt((@x - point.x) ** 2 + (@y - point.y) ** 2)
+  end
+
+  def middle(point)
+    @mid_x = ((@x + point.x).to_f) / 2
+    @mid_y = ((@y + point.y).to_f) / 2
+    puts "#{@mid_x} ; #{@mid_y}"
+  end
+end
+
+class Triangle
+  attr_accessor :a, :b, :c
+
+  def initialize(a, b, c)
+    @a = a
+    @b = b
+    @c = c
+  end
+
+  def triangle_perimeter
+    @ab = @a.distance(@b)
+    @bc = @b.distance(@c)
+    @ac = @a.distance(@c)
+    @perimeter = @ab + @ac + @bc
+    
+  end
+
+  def triangle_isoscel
```

Comment:
đối với method trả về true/false nên thêm `?` vào tên method
```suggestion
  def triangle_isoscel?
```